app/scripts/lib/constants.js

```diff
@@ -24,6 +24,9 @@ define([], function () {
 
     OAUTH_CODE_LENGTH: 64,
 
+    DERIVED_KEY_LENGTH: 32,
+    DERIVED_KEY_INFO_PREFIX: 'identity.mozilla.com/picl/v1/oauth/',
```

Comment:
I propose re-naming them to "RELIER_KEYS_XXX" in this PR for a little more clarity, and we can re-consider the overall factoring in a follow-up issue.